src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,22 @@
+import { useContext } from 'react';
+import { ITodo } from '../../types/ITodo';
+import { AppContext } from '../AppProvider/AppProvider';
+import { Todo } from '../Todo';
+
+type Props = {
+  todos: ITodo[]
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const { tempTodo } = useContext(AppContext);
+
+  return (
+    <>
+      {todos.map(todo => (
+        <Todo key={todo.id} todo={todo} />
+      ))}
+
+      {tempTodo && <Todo key={0} todo={tempTodo} />}
```

Comment:
You don't need a key for this your not displaying it with map

Reply:
yup, I don't see the reason for having key here